cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -0,0 +1,388 @@
+/*
```

Comment:
There is still one issue that we need to support at the framework level. Basically if you look into generated file below (generated.go), it doesn't compile.
It's because "rawType" doesn't take into consideration in which package it is producing code. So even if we are referencing ab object from package "pkg" inside "pkg" package, it will still use "pkg.type" instead of just "type".

@lavalamp FYI